rubocop.yml

```diff
@@ -1,6 +1,10 @@
 # Override default Rubocop confg
 # See https://github.com/bbatsov/rubocop
 
+AllCops:
+  NewCops: enable
+  SuggestExtensions: false
```

Comment:
Silence cumbersome logs, and use the new cops. (Ran once in OTP-MASTER, none of them are conflicting with our practices from what I can see...)